### PR TITLE
Sorting entries in the English TOML file

### LINKED ISSUE #38257: Wrong order for entries in `data/i18n/en/en.toml`
**This is a Bug Report**

**Problem:**
The file `data/i18n/en/en.toml` in the website source code contains localization strings, that should be sorted.

The following keys are at the end of the file, but should actually appear much earlier so that they sort correctly:
* `layout_docs_glossary_architecture_name`
* `layout_docs_glossary_architecture_description`
* `layout_docs_glossary_community_name`
* `layout_docs_glossary_community_description`
* `layout_docs_glossary_core-object_name`
* `layout_docs_glossary_core-object_description`
* `layout_docs_glossary_extension_name`
* `layout_docs_glossary_extension_description`
* `layout_docs_glossary_fundamental_name`
* `layout_docs_glossary_fundamental_description`
* `layout_docs_glossary_networking_name`
* `layout_docs_glossary_networking_description`
* `layout_docs_glossary_operation_name`
* `layout_docs_glossary_operation_description`
* `layout_docs_glossary_security_name`
* `layout_docs_glossary_security_description`
* `layout_docs_glossary_storage_name`
* `layout_docs_glossary_storage_description`
* `layout_docs_glossary_tool_name`
* `layout_docs_glossary_tool_description`
* `layout_docs_glossary_user-type_name`
* `layout_docs_glossary_user-type_description`
* `layout_docs_glossary_workload_name`
* `layout_docs_glossary_workload_description`

**Proposed Solution:**
Move those keys

**Additional Information:**
/language en
/kind cleanup
/good-first-issue

### PR DESCRIPTION
**Pull request by QuinceyJames**
Sorted the entries in the English TOML file in dictionary order

Fixes #38257

### CODE REVIEW

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -148,30 +148,102 @@ other = "Latest Release:"
 [latest_version]
 other = "latest version."
 
-[layouts_blog_pager_prev]
```
Comment (divya-mohan0209) [inline data/i18n/en/en.toml:151]: Why was this moved, please?

File: data/i18n/en/en.toml [hunk 2]
```diff
@@ -148,30 +148,102 @@ other = "Latest Release:"
...
+[layout_docs_glossary_user-type_name]
+other = "User Type"
+
+[layout_docs_glossary_workload_description]
+other = "Applications running on Kubernetes."
+
+[layout_docs_glossary_workload_name]
+other = "Workload"
 
 [layouts_blog_pager_next]
```
Comment (divya-mohan0209) [inline data/i18n/en/en.toml:223]: This should be moved up along with layouts_blog_pager_prev to the correct location BEFORE layout_docs_*
```suggestion
[layouts_blog_pager_next]
```

File: data/i18n/en/en.toml [hunk 3]
```diff
@@ -148,30 +148,102 @@ other = "Latest Release:"
...
 other = "Next >>"
 
+[layouts_blog_pager_prev]
+other = "<< Prev"
+
 [layouts_case_studies_list_tell]
```
Comment (divya-mohan0209) [inline data/i18n/en/en.toml:229]: Needs to be moved up and placed after layouts_blog_pager_* BEFORE layout_docs_*
```suggestion
[layouts_case_studies_list_tell]
```

File: data/i18n/en/en.toml [hunk 4]
```diff
@@ -148,30 +148,102 @@ other = "Latest Release:"
...
 other = "Tell your story"
 
 [layouts_docs_glossary_aka]
```
Comment (divya-mohan0209) [inline data/i18n/en/en.toml:232]: Needs to be at the top of the list for layouts_docs_glossary_*

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I've left a few comments, but this isn't alphabetically sorted yet. Please reorder correctly & resubmit for review.

Thanks!

File: data/i18n/en/en.toml (hunk 2 above)
Comment (QuinceyJames) [reply data/i18n/en/en.toml:223]: layouts_blog_pager_prev comes after layout_docs_*

Should I change them to all to 'layouts' and update the code accordingly?

File: data/i18n/en/en.toml (hunk 3 above)
Comment (QuinceyJames) [reply data/i18n/en/en.toml:229]: See above

File: data/i18n/en/en.toml (hunk 4 above)
Comment (QuinceyJames) [reply data/i18n/en/en.toml:232]: See above

File: data/i18n/en/en.toml (hunk 1 above)
Comment (divya-mohan0209) [reply data/i18n/en/en.toml:151]: Ah, my bad. It is in the correct order then.

**Comment by divya-mohan0209** [comment]
@QuinceyJames Please sign the CLA

**Comment by QuinceyJames** [comment]
CLA Signed!

I am wondering if it would be beneficial for me to update the keys to all be 'layouts' to avoid future confusion?

Also, if you think it would help, I would be more than happy to develop a pre-commit check or a github action that verifies all the keys are in order!

File: data/i18n/en/en.toml (hunk 2 above)
Comment (sftim) [reply data/i18n/en/en.toml:223]: I wouldn't do that change in _this_ PR, as it's a tricky change to get right. We'd want to coordinate with multiple localization teams and it will take several weeks _minimum_. However, please **do** consider filing an issue about how the two prefixes don't match, so we can track the work.

**Comment by kbhawkey** [comment]
👋 @QuinceyJames .
PR #39013 merged and contains the updates to the TOML file.
I'll close this pull request, but feel free to reopen it if you have questions.
Thanks
/close
